apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -90,6 +101,14 @@ export default function TemplateEditorPage() {
     openTestWorkflowModal();
   };
 
+  const onSubmitHandler = async (data: IForm) => {
+    await onSubmit(data, {
+      onCreateSuccess: () => {
+        setTriggerModalVisible(true);
```

Comment:
after the successfull notification template creation show the trigger modal, when it's closed it will redirect to `/templates` page